app/views/layouts/application.html.erb

```diff
@@ -13,6 +13,7 @@
       <% flash.each do |key, value| %>
         <div class="alert alert-<%= key %>"><%= value %></div>
       <% end %>
+      <%= render partial: 'shared/user_bar' %>
```

Comment:
I believe you dont need to specify partial here.

from inside a view file rails automatically assumes it will be a partial.

<%= render shared/user_bar' %>